packages/types/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@artilleryio/types",
+  "version": "0.0.0",
+  "files": [
+    "./definitions.ts",
+    "./schema.json"
+  ],
+  "scripts": {
+    "test": "tap --ts --no-coverage test/**/*.test.ts",
```

Comment:
Can you add `--color` to the script to force colored output in CI? Otherwise in Github Actions it will not use color, which makes the default reporter `raw` rather than the proper reporter. So it looks like this:

![Screenshot 2023-08-03 at 10 26 59](https://github.com/artilleryio/artillery/assets/39738771/087539b3-a568-4a6e-a0ef-b87b02eb1ebe)

Reply:
Done in 90279299e74c6044622cc4e6ff22b7ded5b47813